README.md

```diff
@@ -113,13 +114,23 @@ rails server
 
```

Comment:
- [ ] On the new category form page there should be an arrow (<) button on the navbar that takes the users back to the categories list page.

  Your Implementation | Requirement
  :---:|:---:
  ![image](https://github.com/M-Anwar-Hussaini/Expense-Management/assets/36542986/8bc709d8-39d5-4233-8cf2-70a0bf27b7f0) | ![image](https://github.com/M-Anwar-Hussaini/Expense-Management/assets/36542986/e7258c3e-f31d-402a-9d1b-81851cdc96da)